agent/eni/pause/pause_unsupported.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!windows
 
 // Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is the copyright missing the year?

Reply:
The agent package's copyright message does not include a year.